osu.Framework/Platform/GameHost.cs

```diff
@@ -814,6 +795,221 @@ public void Run(Game game)
             }
         }
 
+        /// <summary>
+        /// The renderer which the game host is currently running with.
+        /// </summary>
+        public RendererType ResolvedRenderer { get; private set; }
```

Comment:
At first glance I thought this is just a duplicate of `IGraphicsSurface.Type`, until I realised it's of a different enum. Probably worth mentioning in remarks section:
```suggestion
        /// <remarks>
        /// This is similar to <see cref="IGraphicsSurface.Type"/> except that this is expressed as a <see cref="RendererType"/> rather than a <see cref="GraphicsSurfaceType"/>.
        /// </remarks>
        public RendererType ResolvedRenderer { get; private set; }
```

Reply:
Or maybe we can just use `Type.ToString()`? It's still giving equal information, just in a slightly different format.